com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -21,6 +21,8 @@ To open the Render Pipeline Debug window in the Editor, go to  **Window > Render
 
 You can display read-only items such as the FPS counter independently of the **Render Pipeline Debug** window. This means that when you disable the **Render Pipeline Debug** window, they are still visible in the top right corner of the screen. This is particularly useful if you want to track particular values without cluttering the screen.
 
+You can disable the runtime UI entirely by using the [enableRuntimeUI](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@10.8/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableRuntimeUI) property.
```

Comment:
Is this URL OK - it doesn't exist right now but I assume it should get generated automatically? Is 10.8 the correct version to begin with?

Reply:
That's a good question. I don't know if there is better way to reference API doc, @Vic-Cooper  any thought?
here the question is how to get the latest doc of a given version (10.x). For this particular case here there is no risk as this API will not change in future 10.x version, but would be good to know if it is something doable.

Reply:
Checked with Vic and the original link is okay. @sebastienlagarde this is good to go on my side.

Reply:
Got it, makes sense. So sounds like xref links would have been OK in this case too. Bit of a bummer though that it's untestable before it goes live.